doc/source/release/template.rst

```diff
@@ -1,45 +1,21 @@
 :orphan:
 
+.. currentmodule:: numpy
+
 ==========================
-NumPy 1.xx.x Release Notes
+NumPy 2.xx.x Release Notes
 ==========================
 
 
 Highlights
 ==========
 
-
-New functions
-=============
-
-
-Deprecations
-============
-
-
-Future Changes
-==============
-
-
-Expired deprecations
-====================
-
-
-Compatibility notes
-===================
-
-
-C API changes
-=============
-
-
-New Features
-============
+*We'll choose highlights for this release near the end of the release cycle.*
```

Comment:
We actually have a highlights section also in the release notes fragments which I think makes sense to not forget a highlight.  OTOH, I am not sure we ever actually used it so this is fine also.

Reply:
Sure, you certainly have to audit it at the end.